api/maven-api-core/src/main/java/org/apache/maven/api/DependencyScope.java

```diff
@@ -92,7 +92,7 @@ public enum DependencyScope {
      * System scope.
      * <p>
      * Important: this scope {@code id} MUST BE KEPT in sync with label in
-     * {@code org.eclipse.aether.util.artifact.Scopes#SYSTEM}.
+     * {@code org.eclipse.aether.util.artifact.DependencyScopes#SYSTEM}.
```

Comment:
we should add a provider spi in resolver, to avoid this "must be KEPT in sync" ...

Reply:
https://issues.apache.org/jira/browse/MRESOLVER-491